### PR TITLE
Add Exaile

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/exaile/exaile/issues/514
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://github.com/exaile/exaile/pull/845 https://github.com/exaile/exaile/pull/846

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build org.exaile.Exaile

**Comment by Mailaender** [comment]
bot, build org.exaile.Exaile

**Comment by Mailaender** [comment]
bot, build org.exaile.Exaile

File: python3-cairo.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (hfiguiere) [inline python3-cairo.json:1]: this manifest builds nothing.

File: python3-py3cairo.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "name": "python3-py3cairo",
```
Comment (hfiguiere) [inline python3-py3cairo.json:2]: this manifest install nothing.

File: python3-pycairo.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "python3-pycairo",
```
Comment (hfiguiere) [inline python3-pycairo.json:2]: there is another manifest fragement that install pycairo.

**Comment by Mailaender** [comment]
bot, build org.exaile.Exaile

File: org.exaile.Exaile.json [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    "python3-gobject.json",
+    "python3-PyGObject.json",
+    "python3-mutagen.json",
+    "python3-dbus-python.json",
+    "python3-feedparser.json",
+    "python3-ipython.json",
+    "python3-pylast.json",
+    "python3-lxml.json",
+    "python3-beautifulsoup4.json",
+    "python3-musicbrainzngs.json",
```
Comment (barthalion) [inline org.exaile.Exaile.json:83]: Any reason not to merge this into a single file? (Also, are ipython and poetry actually required on runtime?)

**Comment by barthalion** [comment]
LGTM, but would prefer to hear back from upstream, given they were initially interested.

File: org.exaile.Exaile.json (hunk 4 above)
Comment (Mailaender) [reply org.exaile.Exaile.json:83]: This is the default output of https://github.com/flatpak/flatpak-builder-tools/tree/master/pip. ipython is (optionally) required for the ipconsole plugin. Removed poetry.

**Comment by Mailaender** [comment]
bot, build org.exaile.Exaile

**Pull request closed without merging**